packages/xod-client/src/editor/actions.js

```diff
@@ -285,8 +289,19 @@ export const copyEntities = event => (dispatch, getState) => {
   )(state);
 
   const entities = getEntitiesToCopy(currentPatch, selection);
-  event.clipboardData.setData(getClipboardDataType(), JSON.stringify(entities, null, 2));
-  event.preventDefault();
+  return Maybe.Just(entities);
+};
+
+export const copyEntities = event => (dispatch, getState) => {
+  if (isInput(event)) return;
+
+  const state = getState();
+
+  // eslint-disable-next-line array-callback-return
+  getClipboardEntities(state).map((entities) => {
```

Comment:
You don’t use the `map` result. Linter is not very happy. Perhaps, it is a job for `forEach`?